src/containers/Profile/selectors.js

```diff
@@ -1 +1,36 @@
+import { getCurrentUserSelector } from 'public-modules/Authentication/selectors';
+import { createSelector } from 'reselect';
+import { isEmpty } from 'lodash';
+
 export const profileUISelector = state => state.profileUI;
+
+const profileSections = [
+  // would there be a difference here between small and large for determining whether they've uploaded a profile pic?
```

Comment:
I'm using `small_profile_image_url` here to see if they've filled the avatar section. I'm guessing there's not a situation where this would be empty despite having an image.

Reply:
that is correct.